### PR TITLE
Character setup ringtone selection

### PR DESCRIPTION
**Pull request by Superlagg**
[FEATURE]
## About the PR 

You can now pick your PDA's default ringtone through character setup!

It also adds the ringtone's corresponding ringtone pack program to your PDA's root drive if there's space, so you can set up your message group ringtones as well.

Also made basic ringtone 10 actually accessible and usable. 

Also updated the null ringtone's description.

## Why's this needed? 

Turns out ringtones aren't used that much! So, someone in discord suggested they might get more use if they could be set in the character settings.

## Changelog

```changelog
(u)Superlagg
(*)You can now pick your PDA's ringtone in character setup! Comes with a free copy of that ringtone's parent program.
```

### CODE REVIEW

**Review by ZeWaka — CHANGES_REQUESTED** [review]
❌

File: code/global.dm [hunk 1]
```diff
@@ -376,6 +376,28 @@ var/global
 	"Yesteryear",
 	"Zeyada")
 
+	list/selectable_ringtones = list("Two-Beep" = new/datum/ringtone,
```
Comment (ZeWaka) [inline code/global.dm:379]: filteredtypesof?

Comment (Superlagg) [reply code/global.dm]: Did that, looks like it works nicely, much better than manually defining them all. Now, availability is defined by a var in the ringtone itself.

File: code/datums/pda_ringtones.dm [hunk 2]
```diff
@@ -5,6 +5,11 @@
 /// Pick a random index
 #define RINGLIST_RANDOM 2
 
+/// type filter for ringtones that're suposed to be selectable at roundstart
+proc/filter_is_character_setup_ringtone(type)
+	var/datum/ringtone/r_tone = type
+	return initial(r_tone.canSpawnWith)
```
Comment (ZeWaka) [inline code/datums/pda_ringtones.dm:11]: hype energy

**Review by ZeWaka — APPROVED** [review]

**Pull request merged**